src/dev/impl/DevToys/Core/QueueWorkerViewModelBase.cs

```diff
@@ -1,20 +1,18 @@
 ﻿#nullable enable
 
 using System.Collections.Generic;
-using System.Threading.Tasks;
-using DevToys.Shared.Core.Threading;
 using Microsoft.Toolkit.Mvvm.ComponentModel;
 
 namespace DevToys.Core
 {
-    public abstract class QueueWorkerViewModelBase<T> : ObservableRecipient where T : class
+    public abstract class QueueWorkerViewModelBase<T> : ObservableRecipient
     {
         private readonly object _lock = new();
         private readonly Queue<T> _computationQueue = new();
 
         private bool _computationInProgress;
 
-        internal Task ComputationTask { get; private set; } = Task.CompletedTask;
+        internal bool ComputationTask { get; private set; }
```

Comment:
Is this property necessary? I don't see it used outside of this class and if so, I feel like the name isn't really good 😅

Reply:
Yeah I forgot to remove it.